packages/grafana-prometheus/src/components/index.ts

```diff
@@ -8,3 +8,4 @@ export * from './PromQueryEditorForAlerting';
 export * from './PromQueryField';
 export * from './types';
 export * from './VariableQueryEditor';
+export * from './PromCheatSheet';
```

Comment:
Probably not something for a follow up PR... for clarities sake, and to prevent accidentally exposing anything you'd like to keep private, I would avoid all nested barrel files entirely and use named exports rather than these wildcards in `src/index.ts`.

Reply:
I can definitely do this!

Reply:
I have removed the nested barrel files I'm exporting with names instead of wildcards. I am aiming to be exporting just enough to make the custom auth work for our external vendor data sources. That will make sure that we aren't exporting anything private for now.